src/poetry/core/_vendor/_pyrsistent_version.py

```diff
@@ -1 +1 @@
-__version__ = '0.19.3'
```

Comment:
Please revert any changes made in `_vendor` directory, those are vendored packages, we don't touch those.

Reply:
> Please revert any changes made in `_vendor` directory, those are vendored packages, we don't touch those.

Will do, too bad it wasn't mentioned anywhere